mvp_example/src/main/java/com/distillery/android/blueprints/mvp/main/contract/TodoContract.kt

```diff
@@ -0,0 +1,17 @@
+package com.distillery.android.blueprints.mvp.main.contract
+
+import com.distillery.android.domain.models.ToDoModel
+
+interface TodoContract {
+    interface View {
+        fun showError(message: String)
+        fun showTaskDeletedMessage()
```

Comment:
Just a possible naming improvement:
```suggestion
        fun notifyTaskDeleted()
```